packages/react-ui/src/app/routes/flows/issues-table/index.tsx

```diff
@@ -19,83 +19,144 @@ import { toast } from '@/components/ui/use-toast';
 import { issuesApi } from '@/features/issues/api/issues-api';
 import { issueHooks } from '@/features/issues/hooks/issue-hooks';
 import { useAuthorization } from '@/hooks/authorization-hooks';
-import { flagsHooks } from '@/hooks/flags-hooks';
 import { authenticationSession } from '@/lib/authentication-session';
 import { useNewWindow } from '@/lib/navigation-utils';
-import {
-  ApEdition,
-  ApFlagId,
-  FlowRunStatus,
-  Permission,
-} from '@activepieces/shared';
+import { FlowRunStatus, IssueStatus, Permission } from '@activepieces/shared';
 
 import { issuesTableColumns } from './columns';
 
 export function IssuesTable() {
   const navigate = useNavigate();
   const { refetch } = issueHooks.useIssuesNotification();
-  const { data: edition } = flagsHooks.useFlag(ApFlagId.EDITION);
-  const isEditionSupported = [ApEdition.CLOUD, ApEdition.ENTERPRISE].includes(
-    edition as ApEdition,
-  );
-  const [searchParams] = useSearchParams();
+
+  const [searchParams, setSearchParams] = useSearchParams();
   const projectId = authenticationSession.getProjectId()!;
+  const prevSearchParamsRef = useRef<string>('');
+  const currentSearchParams = searchParams.toString();
+
+  const statusValues = searchParams.getAll('status');
+  const hasStatusFilter = statusValues.length > 0;
+
+  useEffect(() => {
+    if (!hasStatusFilter) {
+      const newSearchParams = new URLSearchParams(searchParams);
+      newSearchParams.append('status', IssueStatus.UNRESOLVED);
+      setSearchParams(newSearchParams);
+    }
+  }, [hasStatusFilter, searchParams, setSearchParams]);
```

Comment:
I think you just need to `useEffectOnce`

![image](https://github.com/user-attachments/assets/aebae9a4-071e-4258-ad84-f347698b5318)

If you unselect this, It will be reverted to
![image](https://github.com/user-attachments/assets/e153d0f3-ed3e-4531-bc4a-36e77c502e01)